src/main/resources/avro/metadata.avdl

```diff
@@ -114,4 +135,42 @@ record Dataset {
 
 }
 
+/**
+An analysis contains an interpretation of one or several experiments.
+(e.g. SNVs, copy number variations, methylation status) together with
+information about the methodology used.
+*/
+record Analysis {
+  /** The analysis UUID. This is globally unique. */
```

Comment:
UUID? Why is this a requirement? IMO this would ideally just follow the same description as other IDs in the API.